homeassistant/components/workday/const.py

```diff
@@ -12,6 +12,8 @@
 DOMAIN = "workday"
 PLATFORMS = [Platform.BINARY_SENSOR]
 
+ATTR_HOLIDAY_NAME = "holiday_name"
```

Comment:
Why would we add this attribute?

Reply:
I oriented myself in other parts, so I added that. Should I do this instead?

```python
 self._attr_extra_state_attributes["holiday_name"] = holiday
```

Reply:
No, I mean why are we adding this attribute? What would a user do with it?

Reply:
You can display the name of today's holiday, e.g. in a markdown card.

Reply:
I was thinking if there was a use-case for actually making it a sensor and not only adding it as an attribute.

Reply:
Wouldn't it make more sense than to have a "holiday" integration like https://github.com/bruxy70/Holidays/ which provides e.g. a calendar entity?

Reply:
Yes. I have thought about it before but never came around to do it. 
I agree that would be a better fit

Reply:
@jrieger you want to change this PR into a calendar entity instead?

Reply:
I'll have a look.

Reply:
Thinking a bit more on this I probably would say both the attribute, a sensor and a calendar entity would be out of scope for this integration as it targets workdays and not holidays. 
Should probably be it's own integration with only a calendar entity for holidays.

Reply:
See https://github.com/home-assistant/core/pull/103795